src/httpClient/httpClient.ts

```diff
@@ -19,10 +20,14 @@ import { ClientException } from '../exceptions/clientException';
 const invalidToken: string = 'Invalid token';
 
 export default class HttpClient {
+  private readonly orionCorrelationIdRoot = 'orion-correlation-id-root';
```

Comment:
[Very minor]

It is used twice? So we could think of having it in a shared file, and maybe even exporting it out (or not to not leak the magic name).

Reply:
Good point, moved into `shared.ts`